packages/types/src/types.ts

```diff
@@ -262,3 +262,5 @@ export const RESET_FILTERS = 'RESET_FILTERS';
 export interface ResetFilters {
     type: typeof RESET_FILTERS;
 }
+
+export const PAGE_SIZE = 20;
```

Comment:
This could be added to the state, as it is a dynamic value that can change when resizing the window.

Reply:
Done thanks